src/app/bread/bread.router.ts

```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
```

Comment:
Bread entity is not part of the challenge, this files should not be included.

Reply:
Done ✅  . Files removed 👍
Thanks.